### DISCUSSION TITLE
proxy and block issues

### DISCUSSION THREAD

**Discussion by topcj** [tnodir/fort#279] [Q&A] [answered]
When I use a proxy and block a certain program, that program can access the Internet.

**Comment by tnodir** [comment, marked as answer]
1. Let's assume that you use local proxy, so its address is 127.0.0.1 and e.g. port 3128
2. When a certain program connects to Internet via proxy, then it connects to 127.0.0.1:3128
3. If you want to filter the 127.0.0.1 address, then:
  - remove (or comment) the 127.0.0.1 from the Options: IP Addresses: Internet Addresses: Exclude (so this address will be not auto-allowed):
![ip](https://github.com/user-attachments/assets/da52c827-8413-4b93-9a9c-b6b492f54faf)
  - turn on the "Filter local addresses" flag in the Options (so the 127.0.0.1 and 255.255.255.255 addresses will be not auto-allowed):
![local](https://github.com/user-attachments/assets/a0fdadea-6246-46a0-8b41-c004dda85f35)

BUT you must be careful with filtering the 127.0.0.1 (loopback) address, because some programs use it for inter-process communication (IPC)!

**Reply by topcj** [reply]
Thank you very much it worked for me
